ProjectSettings/ProjectSettings.asset

```diff
@@ -68,6 +68,12 @@ PlayerSettings:
   androidRenderOutsideSafeArea: 1
   androidUseSwappy: 1
   androidBlitType: 0
+  androidResizableWindow: 0
+  androidDefaultWindowWidth: 1920
+  androidDefaultWindowHeight: 1080
```

Comment:
fyi, our target resoulution is not 1080p, it's larger - 1125 or smth,
not sure if relevant here